users/src/routes/user.routes.ts

```diff
@@ -36,4 +40,12 @@ router.post(
   userController.manageStatus
 );
 
+router.post(
```

Comment:
this should belong to the team route?

Reply:
there's two routes of manageStatus
one for user: accepting invitation
one for team: accepting request

Reply:
for the user, it still related to teams in my opinion